prometheus/base/alerting/prometheus.controller_alerts.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-controller-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: controller_alerts
+    rules:
+    - alert: ControllerReconciliationErrors
+      expr: |
+        sum(increase(controller_runtime_reconcile_errors_total{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[2m]))
+        by(container, controller, namespace, pod, service) /
+        sum(increase(controller_runtime_reconcile_total{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[2m]))
+        by(container, controller, namespace, pod, service) > 0.01
+      for: 15m
```

Comment:
I suspect by using 15m, it will fail to pickup bursts of errors (see my other comment in the tests). It would be nice to detect those of course. Setting this to `2m` might help but I haven't considered the possible downsides. Wdyt?

Reply:
@amisstea I have updated to 2m and accordingly test cases as well. Please review again.